input-output-hk/daedalus
# [DDW-1000] Integrate legacy wallet transactions list V2 API endpoint

This PR introduces the Legacy Wallet transactions list API V2 endpoint.

## Todos

- [x] Display UNRESOLVED transaction inputs
- [x] Japanese Translations for UNRESOLVED transaction inputs

## Screenshots

_No Addresses_

![EN - No Addresses](https://user-images.githubusercontent.com/18653950/68323689-40dd7000-00c6-11ea-8b86-c5ae7e59188f.png)
![JA - No Addresses](https://user-images.githubusercontent.com/18653950/68323690-40dd7000-00c6-11ea-8155-dc26670ffbc7.png)


_Unresolved addresses_

![EN - Unresolved addresses](https://user-images.githubusercontent.com/18653950/68323719-4d61c880-00c6-11ea-890e-5c0e94f09725.png)
![JA - Unresolved addresses](https://user-images.githubusercontent.com/18653950/68323720-4d61c880-00c6-11ea-84e7-c0c0c3d2f287.png)


---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1573063083355100)

**NOTE: **
_It is not possible to test Legacy wallet transactions because we don't have wallet with transactions_

**1. Test Storybook stories**
- [ ] Test Storybook story under: `WalletTransactionsList -> transactions with unresolved income addresses`
- [ ] Test Storybook story under: `WalletTransactionsList -> transactions without income addresses`

**2. In-App testing**
- [ ]  Transactions without income addresses
- Run `yarn v2-api-importer` to get wallets with funds
- Check transaction details on one off wallets and `From` address should be: `No Addresses`

- [ ]  Transactions with unresolved income addresses 
- Run `yarn v2-api-importer` to get wallets with funds
- Create `Test Wallet`
- Send ADA from one of imported wallets to `Test Wallet`
- Check `Test Wallet` transaction details and `From` address should be: `Open this transaction in Cardano explorer` link.

---

## Review Checklist

### Basics


### Code Quality

### Testing

### After Review
